### PR TITLE
Feature 1 is completed

### PR DESCRIPTION
**Pull request by jamithiprudhvi**
some lines were added

### CODE REVIEW

**Review by kumargomda — DISMISSED** [review]

**Comment by jamithiprudhvi** [comment]
done

**Review by kumargomda — APPROVED** [review]

**Review by prudhvi9395 — APPROVED** [review]

**Comment by prudhvi9395** [comment]
merg completed

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,11 @@
 ### GIT
-This is first commit from main branch
\ No newline at end of file
+This is first commit from main branch
+
```
Comment (kumargomda) [inline README.MD:3]: is this line required

**Review by prudhvi9395 — COMMENTED** [review]
ok

**Review by prudhvi9395 — COMMENTED** [review]
ok

**Pull request merged**